flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/CEPITCase.java

```diff
@@ -361,17 +351,18 @@ public String select(Map<String, Event> pattern) {
 			}
 		);
 
-		result.writeAsText(resultPath, FileSystem.WriteMode.OVERWRITE);
+		result.addSink(resultSink);
```

Comment:
How about using `DataStreamUtils#collect()` here? Then we wouldn't need to move that sink class around.